src/Widgets/DataCountWidget/DataCountWidgetComponent.tsx

```diff
@@ -49,8 +49,11 @@ provideComponent(DataCountWidget, ({ widget }) => {
   )
 })
 
-const Count = connect(
-  function Count({ widget }: { widget: DataCountWidgetInstance }) {
+const Count = connect<
+  { widget: DataCountWidgetInstance },
+  { widget: DataCountWidgetInstance }
```

Comment:
* [ ] FYI first time I see the types used this way. I wonder if this is supported/intended/documented use. Looks weird to have to specify the (seemingly?) same type twice. How about sticking with (see, e.g., DataImageWidget) the more intuitive pattern:

```tsx
const Count = connect(
  function Count({ widget }: { widget: DataCountWidgetInstance }) {
```